README.md

```diff
@@ -71,5 +105,8 @@ You are free to fork this repo and make pull requests to enhance the functionali
 
 Thanks, Ikechi Michael I.
 
+[@allengblack](https://twitter.com/allengblack) made some significant updates to this. Follow him on Twitter, too and you can see some more of his work [here](https://github.com/allengblack)
```

Comment:
Not a deal breaker, but clicking @allengblack already links to your GitHub, so you can add your twitter in the here link